### ISSUE TITLE
macos make fails on simple node commands

### ISSUE THREAD

**Issue by Pike** [projectfluent/fluent.js#486]
macos is super eager to not pass commands to the shell at all, if it thinks they're simple.

Thus the PATH logic doesn't work for things like `tsc`, or *some* invocations of eslint. Yes, *some*.

There are at least three ways to fix these:

```
lint-1:
    @: ; eslint --config $(ROOT)/eslint_test.json --max-warnings 0 test/
lint-2:
    @eslint --config $(ROOT)/eslint_test.* --max-warnings 0 test/
lint-3:
    @../.node_modules/.bin/eslint --config $(ROOT)/eslint_test.json --max-warnings 0 test/
```

`common.mk` could probably also define `ESLINT` and `TSC` to the path in `lint-3`, which would double-fix the issue, probably, as the shell variable would trigger the shell, and it'd be a real path.

@stasm, opinions?

**Comment by stasm** [comment]
I'm leaning towards option no. 3: `../.node_modules/.bin/eslint `. It's simple (as in _the opposit of _indirect_), explicit, self-explanatory, and the visual noise isn't much of a problem, because we don't spend too much time looking at Makefiles anyways. I can work on a PR if you agree.

**Comment by Pike** [comment]
SGTM. I'll make a patch, 'cause I need to test which things pass and which fail anyway

**Comment by Pike** [comment]
This is fixed by #487
